internal/scaffold/role_test.go

```diff
@@ -82,6 +86,336 @@ func TestRoleCustomRules(t *testing.T) {
 	}
 }
 
+func TestMergeHelmRoleForResource(t *testing.T) {
+	RoleFilePath1 := "./testdata/testroles/valid_role1"
+	absRoleFile1 := RoleFilePath1 + "/deploy/role.yaml"
+	if err := ioutil.WriteFile(absRoleFile1, []byte(roleFile1), fileutil.DefaultFileMode); err != nil {
+		fmt.Printf("failed to update %v: %v", absRoleFile1, err)
+	}
+
+	RoleFilePath2 := "./testdata/testroles/valid_role2"
+	absRoleFile2 := RoleFilePath2 + "/deploy/role.yaml"
+
+	if err := ioutil.WriteFile(absRoleFile2, []byte(roleFile2), fileutil.DefaultFileMode); err != nil {
+		fmt.Printf("failed to update %v: %v", absRoleFile2, err)
+	}
```

Comment:
Ditto: can we check these in or make sure to clean them up?

Reply:
The reason why I skipped cleaning up, is that in the beginning of processing the test cases, I am writing content to the role files. That way, at the beginning of test cases, it will always have data from `const`. However, I do understand that when someone takes a look at the `testdata` from code, it will always have `original` and `merged` files same, as that is the result at the end of test case. Since we dont cleanup, they stay the same way, until testcases are run again. Does that make sense?

Reply:
Added `defer remove`